iotedgehubdev/errors.py

```diff
@@ -39,29 +39,26 @@ def __init__(self, msg, ex=None):
         super(EdgeDeploymentError, self).__init__(msg, ex)
 
 
-class ResponseError(Exception):
+class ResponseError(EdgeError):
     def __init__(self, status_code, value):
+        super(ResponseError, self).__init__(value)
```

Comment:
status_code not used? So the message won't have status code?

Reply:
updated